### PR TITLE
[Feedback] Fix breakpoint for medium devices

### PR DESCRIPTION
**Pull request by nerdlet**
##### What does this PR do?
=>Fix responsiveness on medium devices ` ipad pro min-width:1024 `
##### Description of Task to be completed?
=>Fixed transition
=>Added opacity to images
=>Fix display on medium devices

##### What are the relevant Pivotal Tracker stories?
 #166838253

##### Screenshots?

image

image

On hover:
image

### CODE REVIEW

File: src/components/OurPartners.js [hunk 1]
```diff
@@ -15,25 +15,45 @@ import Fisheries from '../assets/fisheries.png';
...
-    width: '240px',
-    height: '360px',
+    height: '19.5rem',
     backgroundColor: '#ffffff',
     boxShadow: '0 30px 60px 0 rgba(0, 0, 0, 0.05)',
     flex: '1',
     textAlign: 'center',
-    margin: '10px',
+    margin: '1rem',
+    transition: 'transform 1s',
```
Comment (kilemensi) [inline src/components/OurPartners.js]: `1s` is a bit too long for this transition...

File: src/components/OurPartners.js [hunk 2]
```diff
@@ -15,25 +15,45 @@ import Fisheries from '../assets/fisheries.png';
...
-    [theme.breakpoints.up('md')]: {
-      width: '400px',
-      height: '400px',
-      margin: '10px 40px'
+    [theme.breakpoints.up('lg')]: {
+      width: '23.4375rem',
+      height: '23.4375rem',
+      margin: '1.5rem'
+    },
+    [theme.breakpoints.width('1024')]: {
```
Comment (kilemensi) [inline src/components/OurPartners.js]: Use breakpoints: `xs`, `sm`, `md`, etc.

Comment (kilemensi) [reply src/components/OurPartners.js]: And these properties here are **exactly** like `lg` above. Why a different `breakpoint` then?

**Review by kilemensi — CHANGES_REQUESTED** [review]
@nerdlet  can you may be:

1. Separate component size _clearly_ into three parts: Mobile, Medium and Large.
2. Ensure the components remain as **square** and have the same size when not hovered in all screen sizes,
2. When hovered/activated, _increase_ the **height** without changing the **width**. From the design we can see Nukta Africa remains the same width and in the same position, just grows taller/longer).

File: src/components/OurPartners.js (hunk 2 above)
Comment (nerdlet) [reply src/components/OurPartners.js]: @kilemensi Removed the `breakpoint 1024`. It wasn't really necessary at this point. Instead, moved the breakpoint from `md` to the `lg` . This fixed the issue on the ` Ipad pros ` and all other medium devices.

**Comment by nerdlet** [comment]
@kilemensi Done. Could you test the `on hover` effect on your end?

**Comment by kilemensi** [comment]
> @kilemensi Done. Could you test the `on hover` effect on your end?

I've pushed a small change on how I think we should do the transition @nerdlet...

Peek 2019-06-24 15-27

**Comment by kilemensi** [comment]
You may increase the height adjustment if you wish @nerdlet but for me the cards are still not square (as you can see from the above gif).

**Comment by nerdlet** [comment]
@kilemensi Okay. Taking a look at the card measurements now

**Comment by nerdlet** [comment]
@kilemensi LGTM...On my end at least. Maybe for the small devices on the carousel, they look a tad bit off...

**Review by karimkawambwa — CHANGES_REQUESTED** [review]
I see alot of px
If using vscode there is nice plugin to help convert them all

```
Name: px to rem
Id: sainoba.px-to-rem
Description: Converts px to rem, and vice versa
Version: 1.2.7
Publisher: Marco N
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=sainoba.px-to-rem
```

From the screeshot, they don't squared and the  below logo text should probably be centered more:

Screen Shot 2019-06-25 at 12 05 46 PM


Otherwise its all gucci 💯 , very very minor stuff.

**Comment by nerdlet** [comment]
@kilemensi Done

**Pull request merged**